.github/workflows/start-release.yml

```diff
@@ -45,6 +52,7 @@ jobs:
           submodules: "recursive"
           ref: "${{ github.event.repository.default_branch }}"
           token: "${{ steps.token.outputs.token }}"
+      #TODO: Why do we need this? checkout@v3 does not get the tags by default?
```

Comment:
See comment below, however, no. checkout@v3 only gets tags if you do a fully checkout, which is too heavyweight. We don't want all branches and everything that ever existed on the repo, so we just do a lightweight checkout and then get the tags ourselves.